docs/content/usage/issue-pull-request-templates.en-us.md

```diff
@@ -19,9 +19,10 @@ menu:
 
 Some projects have a standard list of questions that users need to answer
 when creating an issue or pull request. Gitea supports adding templates to the
-main branch of the repository so that they can autopopulate the form when users are
+**main branch of the repository** so that they can autopopulate the form when users are
```

Comment:
not "default branch"?

Reply:
yes looks better to use default as wording